plugins/kubernetes/app/models/kubernetes/template_filler.rb

```diff
@@ -31,8 +31,7 @@ def to_hash
             set_history_limit
           end
 
-          set_replica_target unless ['DaemonSet', 'Pod'].include?(kind)
-
+          set_replica_target unless ['DaemonSet', 'Pod'].include?(template[:kind])
```

Comment:
shouldn't this be checking the autoscale thing and not do it if the role is autoscaled?

Reply:
the "set replicas" logic is in resource.rb where the actual update is being done, the template does not know if it is being used for a create or update and I wanted the create to still have the user-defined number of replicas